File: app/src/main/java/awais/instagrabber/models/enums/BroadcastItemType.java
package awais.instagrabber.models.enums;

public enum BroadcastItemType {
    TEXT("text"),
    REACTION("reaction"),
    REELSHARE("reel_share"),
    IMAGE("configure_photo"),
    LINK("link"),
    VIDEO("configure_video"),
    VOICE("share_voice");

    private final String value;

    BroadcastItemType(final String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: app/src/main/java/awais/instagrabber/models/enums/BroadcastItemType.kt
package awais.instagrabber.models.enums

enum class BroadcastItemType(val value: String) {
    TEXT("text"),
    REACTION("reaction"),
    REELSHARE("reel_share"),
    IMAGE("configure_photo"),
    LINK("link"),
    VIDEO("configure_video"),
    VOICE("share_voice"),
    ANIMATED_MEDIA("animated_media"),
    MEDIA_SHARE("media_share"),
}